tests/test_meshtastic_utils_edge_cases.py

```
@@ -375,18 +379,23 @@ def test_on_meshtastic_message_plugin_timeout_uses_config(self):
         }
         rooms = [{"meshtastic_channel": 0, "id": "!room:matrix"}]
 
+        def _submit_with_timeout_future(coro, loop=None):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_submit_with_timeout_future`

(ANN202)